infrastructure/sentry-subgraph/src/pool-factory.ts

```diff
@@ -109,26 +109,20 @@ export function handleStakeKeys(event: StakeKeys): void {
 
   let nodeLicenseIds: BigInt[];
 
-  const signature = getTxSignatureFromEvent(event);
-
-  // Check if this is triggered from the tiny keys airdrop admin stake 
-  // processAirdropSegmentOnlyStake(uint256) => 0x3ada44c1
-  if (signature == "0x3ada44c1") {
-
-    // If the event was triggered by the airdrop admin stake, we ignore as that event is handled separately
-    return;  
-  
-  } else {
-    const dataToDecode = getInputFromEvent(event, true)
-    const decoded = ethereum.decode('(address,uint256[])', dataToDecode);
+    const dataToDecode = getInputFromEvent(event, true);
+    const decoded = ethereum.decode("(address,uint256[])", dataToDecode);
```

Comment:
?

Reply:
Looks like the formatter changed single quotes to double quotes.

Reply:
Neat. We should do a PR with the formatting I believe.